pkg/features/versioned_kube_features.go

```diff
@@ -559,6 +559,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	PodIndexLabel: {
 		{Version: version.MustParse("1.28"), Default: true, PreRelease: featuregate.Beta},
+		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.35
 	},
```

Comment:
```suggestion
		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.34
	},
```
I think 2 releases is enough as we did here: https://github.com/kubernetes/kubernetes/pull/128387/files#diff-530f68e01a51f788bd0ad558bde87a4a378bd3e40e3811d097029f6aa109c7a7R551

Reply:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/4330-compatibility-versions/README.md#feature-gate-lifecycles

This has changed in this release with the emulated version KEP. Removal is 3 releases after GA now.

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md#compatibility-versions

Reply:
I see, thanks for the pointer.